docs/sql/kill.txt

```diff
@@ -0,0 +1,36 @@
+.. highlight:: psql
+.. _kill_job:
+
+=========
+Kill jobs
+=========
+
+Each request in Crate that fetches or manipulates data is called "job".
+Jobs that are currently executed are logged in the system table ``sys.jobs`` 
+(see :ref:`jobs_operations_logs`).
+
+Killing an active job forces Crate to stop its execution on the cluster 
+immediately. There are two different SQL commands available for killing jobs.
+
+The ``KILL ALL`` statement stops every single job on each node that is 
+currently running. It returns the number of job-contexts that have been killed 
```

Comment:
It returns the total number of contexts of all jobs of that have been killed. A job can have contexts on multiple nodes.